extensions/VisualEditor/wikia/modules/ve/ve.track.js

```diff
@@ -175,6 +176,18 @@ require( ['wikia.tracker'], function ( tracker ) {
 			return;
 		}
 
+		if ( ve.init.wikia.getToolbarABTestVariantNumber() === 1 && data.label === 'edit-page-ready' ) {
```

Comment:
There is a custom variable for tracking what happens in the A/B test, see trackingLabel further up in this file. This takes care of everything asked for in the issues except for this requirement:

editor-ve // impression // edit-page-ready - already being tracked in GA but we need a way to bucket this data to A or B groups

This is the reason I send the extra edit-page-ready-toolbartest event in addition to edit-page-ready. Initially, I modified the edit-page-ready event, but in the original PR you advised me to send both.